Extensions/DotnetCoreCLIPackage/Src/readme.md

```diff
@@ -0,0 +1,21 @@
+# Bitbucket&reg; artifacts for Release management
```

Comment:
Bitbucket?

Reply:
The read me got copied from Bitbucket. Will remove this.